code/modules/mob/living/simple_animal/abnormality/waw/dreaming_current.dm

```diff
@@ -132,13 +140,15 @@
 		potential_turfs -= T
 	if(!LAZYLEN(movement_path))
 		return FALSE
-	icon_state = "current_prepare"
+	if(!alt_icon)
```

Comment:
Does it not shoot bubbles if it's in alt-icon?

Reply:
iirc it should still do so, it just doesnt change the iconstate. the bubbles still appear  as normal